Plugins/AWSLambdaPackager/Plugin.swift

```diff
@@ -202,8 +202,27 @@ struct AWSLambdaPackager: CommandPlugin {
             try FileManager.default.copyItem(atPath: artifactPath.string, toPath: relocatedArtifactPath.string)
             try FileManager.default.createSymbolicLink(atPath: symbolicLinkPath.string, withDestinationPath: relocatedArtifactPath.lastComponent)
 
+            // add resources
+            let contentsDirectory = workingDirectory.appending("Contents")
```

Comment:
do we really need a `Contents` directory ? Maybe `Resources` is enough

Reply:
After some more testing the correct bundle to use is `Bundle.module`. So instead of recreating the `Contents` directory I am just copying the `.resources` directory that is in the artifacts directory to the working directory to be zipped. This has the benefit of working on Lambda as well as when running locally on macOS.

Reply:
https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package#Access-a-resource-in-code